_episodes/10-collaborating.md

```diff
@@ -39,45 +40,58 @@ Next, the Collaborator needs to download a copy of the Owner's repository to her
 her `Desktop` folder, the Collaborator enters:
 
 ~~~
-$ git clone https://github.com/vlad/planets.git ~/Desktop/vlad-planets
+$ git clone https://github.com/ccline/cats-as-data.git ~/Desktop/ccline-cats-as-data
 ~~~
 {: .bash}
 
-Replace 'vlad' with the Owner's username.
+Replace `ccline` with the Owner's username. Also note the username prefix on the directory name. This is done because
+you already have your own project named `cats-as-data` in the Desktop folder.
 
 ![After Creating Clone of Repository](../fig/github-collaboration.svg)
```

Comment:
I guess we can't avoid the old repo name / user name in this one?

Reply:
Actually it's SVG so maybe we can 😈

Reply:
brilliant! hacking it now.

Reply:
better? https://github.com/ucsdlib/git-novice/blob/768f6cb1eb02d39f7294e8937f8594aa38b6ff90/fig/github-collaboration.svg

Also took the opportunity to update it with a more accurate path to reflect what we're having them do.

Reply:
yesssss